### DISCUSSION TITLE
Servers not saved between sessions

### DISCUSSION THREAD

**Discussion by rflois** [openscopeproject/TrguiNG#50] [Q&A] [answered]
I'm running TrguiNG on my Mac, OS 13.4.1. If I quit the app, it loses my configured server. Not sure why this might be happening. Is it supposed to save these settings in some config file? Confirmed the issue with 0.6.0 and 0.7.0.

**Comment by qu1ck** [comment]
Yes, a config file is supposed to be created when the app is closed or hidden, on mac it should be in 

`$HOME/Library/Application Support/trguing.json`

**Reply by rflois** [reply]
Got it. File is there, but not being written to when I use Command+Q or choose **Quit TrguiNG**. If I use Command+W, or choose **Close Window** from the File menu, saving works just fine. It also appears that the Save button in the settings doesn't actually write to the config file, only closing the program as described above using **Close Window**.

**Reply by qu1ck** [reply]
Hm, maybe macos has some special case for shutting down apps with cmd+q, I'll look into it.
But otherwise what you are describing is intended.

**Reply by qu1ck** [reply]
Fix for cmd+q committed, will be in next version.

**Reply by qu1ck** [reply, marked as answer]
Fix is in [v0.8.0](https://github.com/openscopeproject/TrguiNG/releases/tag/v0.8.0).

**Comment by rflois** [comment]
Confirmed it's working for me. Thanks @qu1ck !
